examples/sim/compute_tile/compute_tile_sim.core

```diff
@@ -80,14 +82,16 @@ parameters:
     default: 1
     description: Specify the number of CPU cores per tile
   ENABLE_PERFCOUNTERS:
-    datatype: bool
+    datatype: int
     paramtype: vlogparam
     scope: public
+    default: 0
     description: Enable performance counters
   ENABLE_FPU:
-    datatype: bool
+    datatype: int
```

Comment:
@lucasandradecb here I'd say that `ENABLE_FPU` should be a boolean value, since it is either enabled or disabled.... What do you thing?

Reply:
@ppenna I tried to run ENABLE_FPU and ENABLE_PERFCOUNTER with data type bool, but it didn't work. When I changed to int, it worked correctly.